apis/config/v1beta1/configuration_types.go

```diff
@@ -123,6 +123,11 @@ type ControllerMetrics struct {
 	// It can be set to "0" to disable the metrics serving.
 	// +optional
 	BindAddress string `json:"bindAddress,omitempty"`
+
+	// EnableClusterQueueResources, if true the cluster queue resource usage and quotas
+	// metrics will be reported.
+	// +optional
+	EnableClusterQueueResources bool `json:"enambleClusterQueueResources,omitempty"`
```

Comment:
```suggestion
	// enableClusterQueueResources, if true the cluster queue resource usage and quotas
	// metrics will be reported.
	// +optional
	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
```

Reply:
changed the json tag, the member comments are using the name it the full config